Project.toml

```diff
@@ -22,6 +22,7 @@ KernelAbstractions = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
+MPICH_jll = "7cb0a576-ebde-5e09-9194-50597f1243b4"
```

Comment:
do we need this dependency?

Reply:
nono, whenever I push from my mac I have changes to the manifest because of `using Pkg; Pkg.add(name="MPICH_jll", version="4.0.1"); Pkg.build()` (still quite annoying). I usually revert back Manifest and Project before merging

Reply:
oh yes, I see why you need that :)